config/initializers/feature_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+if Rails.version >= "6.1"
```

Comment:
Is it worth checking that the method is available in case this feature doesn't make it into the 6.1 release?
```ruby
if Rails.version >= "6.1" && Rails.application.config.respond_to?(:feature_policy)
```

Reply:
Hi @reggieb - great suggestion thanks 👍  - have updated as per your suggestion.